site/content/en/docs/formats/mpii.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'MPII Human Pose Dataset'
+linkTitle: 'MPII Human Pose Dataset'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original MPII Human Pose Dataset is available
+[here](http://human-pose.mpi-inf.mpg.de/#overview).
+
+Supported annotation types:
+- `Bbox`
+- `Points`
+
+Supported attributes:
+- `center` (list)
+- `scale` (float)
+
+## Import MPII Human Pose Dataset
+
+Datumaro does not support `MATLAB` annotation files. Instead,
+`JSON` and `NUMPY` files are supported.
```

Comment:
Some context is needed here. Why are we talking about MATLAB?

The inline code style also seems to be misapplied here - these aren't identifiers.

Reply:
The original dataset has MATLAB files (`.mat`). These files have been replaced with NUMPY and JSON.

> The inline code style also seems to be misapplied here - these aren't identifiers.

I don't understand how exactly this needs to be fixed.

Reply:
> These files have been replaced with NUMPY and JSON.

Replaced by whom?

> I don't understand how exactly this needs to be fixed.

By removing backticks.

Reply:
> Replaced by whom?

In our case, it comes from AC. AC does not have the ability to read matlab files (on individual platforms).
Also, as it seemed to me, in other datasets, matlab files are often replaced with JSON.

Reply:
Okay, but then you need to provide instructions for the user on how to convert the annotation files into the supported format.

Reply:
Also, what about the original format then? Should we support it in another format or in this one? MATLAB files can be perfectly read from python with [scipy or HDF5](https://stackoverflow.com/a/19340117).